### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Koliras**
[DEMO LINK](https://Koliras.github.io/layout_landing-page/)

### CODE REVIEW

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Good job! Some comments:
- add favicon to your page
- Add some styles on hover state for the links near the contact form
- All fields in the form should be required, for now I don't see any error when I'm trying to submit an empty form
- Will be great to limit the size for the images on the small-medium screens, looks too large for now (https://prnt.sc/OAqgXLe3_Zyn)

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Did you fix previous comments? I don't see favicon

**Comment by Koliras** [comment]
> Did you fix previous comments? I don't see favicon

Yep. Just checked once more, locally everything works. And that's strange, since everything should work corectly on github either. Previously I had already had such problem, when I was practicing in writing code besides mate, but I don't know how to fix it

Here is the proof, that locally favicon doesn't disappear
https://imgur.com/a/anaPMKY

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,263 @@
...
+      rel="icon"
+      type="image/x-icon"
+      href="./images/B_O_Black.svg"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
```
Comment (dffUqp) [inline src/index.html:16]: remove this comment

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,263 @@
...
+  <body class="page__body">
+    <header class="header">
+      <div class="top-bar">
+        <a href="#" class="top-bar__logo-link">
+          <img src="./images/logo.png"
+            alt="logo"
+            class="top-bar__logo">
```
Comment (dffUqp) [inline src/index.html:23]: code style

File: src/styles/blocks/product.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    margin: 0 0 24px;
+    // max-width: 450px;
+
+    // @include onTablet {
+    //   max-width: 800px;
+    // }
+
+    // @include onDesktop {
+    //   max-width: none;
+    // }
```
Comment (dffUqp) [inline src/styles/blocks/product.scss:26]: remove this comments

**Review by dffUqp — CHANGES_REQUESTED** [review]
I see that's code is good, but I can't check it because your demo link is not working:upside_down_face:

**Review by dffUqp — APPROVED** [review]
GJ🚀

**Pull request closed without merging**